BoxContentSDK/BoxContentSDKTests/BOXRequestTests.m

```diff
@@ -222,8 +262,6 @@ - (void)test_that_forbidden_403_error_does_not_trigger_logout_notification_for_n
         [expectation fulfill];
     }];
     [self waitForExpectationsWithTimeout:2.0 handler:nil];
-
-    [operationMock verify];
```

Comment:
NOTE - this was removed because it was unnecessary. The `reject` call above sets up a fail-fast scenario, when the test is failed immediately if `sendLogoutNotification` is called. And there are no other OCMock expectations... thus calling verify is a no-op. So I removed. All this is unrelated to fixing the flakiness of this test.